src/composables/useLocalStorage.js

```diff
@@ -1,5 +1,18 @@
 import { ref } from 'vue'
 
+/**
+ * Function for initialising, adding to and removing from local storage.
+ * Values are stored in arrays with and can be accessed
+ * via a key which specified by the 'key' parameter.
+ * The local storage array is initialized with a value
+ * specified by the 'initialValue' parameter.
+ * Note: this could be replaced by a VueUse composable in the future
+ * https://vueuse.org/core/useLocalStorage/
+ *
+ * @param {string} key
+ * @param {*} initialValue
+ * @returns {function}
```

Comment:
This is not correct - it returns an object with the fields `itemOnLoad, item, ...` but I wouldn't worry about including that info right now
```suggestion
```

Reply:
Ive added some JSdocs for that composable object